src/SmartComponents/CompletedTaskDetails/JobResultsDetails/EntryDetails.js

```diff
@@ -75,11 +65,21 @@ const EntryDetails = ({ entry, taskConstantMapper }) => {
     return (
       <React.Fragment>
         {createGrid('Summary', summary)}
-        {detail?.diagnosis?.context
-          ? createGrid('Diagnosis', renderDiagnosisDetails)
+        {detail?.diagnosis && detail?.diagnosis[0]?.context !== ''
```

Comment:
```suggestion
        {detail?.diagnosis && detail?.diagnosis?.[0]?.context !== ''
```
otherwise, this optional chaining can break if `.diagnosis` is not an array